src/set_theory/pgame.lean

```diff
@@ -904,6 +925,9 @@ theorem add_congr {w x y z : pgame} (h₁ : w ≈ x) (h₂ : y ≈ z) : w + y 
  calc x + z ≤ x + y : add_le_add_left h₂.2
         ... ≤ w + y : add_le_add_right h₁.2⟩
 
+theorem sub_congr {w x y z : pgame} (h₁ : w ≈ x) (h₂ : y ≈ z) : w - y ≈ x - z :=
+add_congr h₁ (neg_congr h₂)
```

Comment:
I hope it's not necessary to have theorems like this. We should just have the bare minimum to prove that `game` is an `add_comm_group`, and then all such equations come for free.